test/RPCSession.cpp

```diff
@@ -166,7 +166,6 @@ RPCSession::TransactionReceipt RPCSession::eth_getTransactionReceipt(string cons
 {
 	TransactionReceipt receipt;
 	Json::Value const result = rpcCall("eth_getTransactionReceipt", { quote(_transactionHash) });
-	BOOST_REQUIRE(!result.isNull());
```

Comment:
Shouldn't we at least `BOOST_CHECK`?

Reply:
We do the same check inside the `rpcCall` function now

Reply:
though... maybe we aren't actually. Maybe it can be member but be `null` too. I should be more precise in my if check